zforce: add page (#10868)
tldr-pages/tldr: pages/linux/zforce.md

# zforce

> > Add a '.gz' extension on all supplied gzip files.

Reviewer: ```suggestion
> Add a GZ extension on all supplied GZip files.
```
For page descriptions, we could go with capitalized extensions/names. (We can leave the examples using quotes to wrap extensions as it is)

> > Add a GZ extension on all supplied GZip files.
> > More information: <https://manned.org/zforce.1>.

Reviewer 2: ```suggestion
> Add a `.gz` extension to files compressed using `gzip`.
> More information: <https://manned.org/zforce>.
```

> - Add a '.gz' extension to the supplied GZip files (Note: Non-GZip files are ignored):

Reviewer: ```suggestion
- Add a `.gz` extension to the supplied GZip files (Note: Non-GZip files are ignored):
```

> - Add a `.gz` extension to the supplied GZip files (Note: Non-GZip files are ignored):

Reviewer 2: ```suggestion
- Add a `.gz` extension to the supplied Gzip files (Note: other files are ignored):
```

> `zforce {{path/to/archive1.gz path/to/archive2.gz ...}}`

Reviewer 2: This won't do anything, will it? The files already end with `.gz`.
```suggestion
`zforce {{path/to/file1 path/to/file2 ..}}`
```

> - Add a '.gz' extension to the supplied gzip files, leave the suppied non-gzip files as-is:
> 
> `zforce {{path/to/gzipped1}} {{path/to/gzipped2}} {{path/to/non/gzipped}}`

Reviewer: ```suggestion
- Add a '.gz' extension to the supplied GZip files (leave the supplied non-gzip files as-is):

`zforce {{path/to/archive1.gz path/to/archive2.gz ...}}`
```
